docs/source/deployorderer/ordererchecklist.md

```diff
@@ -0,0 +1,307 @@
+* **`ServerInterval`**: (default value should not be overridden, unless determined necessary through testing)
+* **`ServerTimeout`**: (default value should not be overridden, unless determined necessary through testing)
+
+## General.Cluster
+
+```
+# SendBufferSize is the maximum number of messages in the egress buffer.
+# Consensus messages are dropped if the buffer is full, and transaction
+# messages are waiting for space to be freed.
+SendBufferSize: 10
+# ClientCertificate governs the file location of the client TLS certificate
+# If not set, the server General.TLS.Certificate is re-used.
+ClientCertificate:
+# If not set, the server General.TLS.PrivateKey is re-used.
+ClientPrivateKey:
+# The below 4 properties should be either set together, or be unset together.
+# If they are set, then the orderer node uses a separate listener for intra-cluster
+# communication. If they are unset, then the general orderer listener is used.
+# This is useful if you want to use a different TLS server certificates on the
+# client-facing and the intra-cluster listeners.
+
+# ListenPort defines the port on which the cluster listens to connections.
+ListenPort:
+# ListenAddress defines the IP on which to listen to intra-cluster communication.
+ListenAddress:
+# ServerCertificate defines the file location of the server TLS certificate used for intra-cluster
+# communication.
+ServerCertificate:
+# ServerPrivateKey defines the file location of the private key of the TLS certificate.
+ServerPrivateKey:
+```
+
+As of Fabric v2.3.0, if unset, the `ClientCertificate` and `ClientPrivateKey` default to the server `General.TLS.Certificate` and `General.TLS.PrivateKey`.
+
+* **`ClientCertificate`**:  Provide the path to and filename of the public certificate (also known as a signed certificate) generated by your TLS CA for this node.
+* **`ClientPrivateKey`**: Provide the path to and filename of the private key generated by your TLS CA for this node.
+
+In general, these four parameters would only need to be configured if you want to configure multiple listeners for your orderers. In most circumstances a single listener is sufficient. Either set these four parameters together or leave them unset.
+
+* **`ListenPort`**
+* **`ListenAddress`**
+* **`ServerCertificate`**
+* **`ServerPrivateKey`**
+
+Note: the `ServerCertificate` and `ServerPrivateKey` must be set to the same values as the `General.TLS.PrivateKey` and `General.TLS.Certificate`.
+
+## General.BoostrapMethod
+
+```
+# Bootstrap method: The method by which to obtain the bootstrap block
+# system channel is specified. The option can be one of:
+#   "file" - path to a file containing the genesis block or config block of system channel
+#   "none" - allows an orderer to start without a system channel configuration
+BootstrapMethod: file
+```
+
+* **`BootstrapMethod`**: (default value should not be overridden) Unless you plan to use a file type other than “file”, this value should be left as is.
```

Comment:
This first cut at the deployment guide describes the system channel process, so let's leave this as is and add the note about the participation API for later. This edit would only confuse people as it would be the only reference to the new channel creation process in the whole guide.

Reply:
ok